neptunecontrib/api/table.py

```diff
@@ -65,10 +66,48 @@ def log_table(name, table, experiment=None):
     _exp.log_artifact(export_pandas_dataframe(table), "tables/" + name + '.html')
 
 
-def export_pandas_dataframe(table):
+def log_csv(name, table, experiment=None):
+    """Logs pandas dataframe to neptune as csv file.
+
+    Pandas dataframe is converted to csv fie and logged to Neptune as an artifact with path csv/{name}.csv
+
+    Args:
+        name (:obj:`str`):
+            | Name of the file (without extension) that will be used as a part of csv's destination.
+        table (:obj:`pandas.Dataframe`):
+            | DataFrame table
+        experiment (:obj:`neptune.experiments.Experiment`, optional, default is ``None``):
+            | Neptune Experiment object if you want to control to which experiment you log the data.
+            | If ``None``, log to currently active, and most recent experiment.
+
+    Examples:
+        Create or load dataframe:
+
+        .. code:: python3
+
+            import pandas as pd
+            iris_df = pd.read_csv('https://raw.githubusercontent.com/mwaskom/seaborn-data/master/iris.csv', nrows=100)
+
+        Log it to Neptune:
+
+        .. code:: python3
+
+             from neptunecontrib.api import log_csv
+             log_csv('pandas_df', iris_df)
+     """
+    _exp = experiment if experiment else neptune
+
+    _exp.log_artifact(export_pandas_dataframe(table, 'csv'), "csv/" + name + '.csv')
```

Comment:
Inconsistent. You mix standard and single quotation marks in one line. It would albo be more clear if you use 'csv/{}.csv'.format(name)

Reply:
corrected.